### PR TITLE
Ym/imp hotel booking test

### LINKED ISSUE #44: Implement hotel booking test

### PR DESCRIPTION
**Pull request by yefim-mironyuk**
PEP8 changes
hotel test implemented

### CODE REVIEW

**Review by kit-91-daniil — APPROVED** [review]
Nice code)

File: tests/conftest.py [hunk 1]
```diff
@@ -153,3 +157,23 @@ def trip_viewer_page_overview_tab(browser):
...
+
+
+@pytest.fixture(scope="module")
+def hotels_list_page(browser):
+    return HotelsListPage(browser=browser)
+
+
+@pytest.fixture(scope="module")
+def hotel_page(browser):
+    return HotelPage(browser=browser)
```
Comment (relaxing-observer) [inline tests/conftest.py:169]: HotelPage/HotelMainPage - a bit confusing

**Review by relaxing-observer — APPROVED** [review]

**Comment by relaxing-observer** [comment]
I suggest making commit names a bit meaningful in future) It's useful in solving conflicts

**Comment by yefim-mironyuk** [comment]
> I suggest making commit names a bit meaningful in future) It's useful in solving conflicts

Yep. I agree. I fcked up a little during rebasing.

**Review by TA-user — APPROVED** [review]

**Pull request merged**